eng/Versions.props

```diff
@@ -253,7 +253,7 @@
       create a test insertion in Visual Studio to validate.
     -->
     <NewtonsoftJsonVersion>12.0.2</NewtonsoftJsonVersion>
-    <StreamJsonRpcVersion>2.4.34</StreamJsonRpcVersion>
+    <StreamJsonRpcVersion>2.6.21-alpha</StreamJsonRpcVersion>
```

Comment:
+1 to Shen's comment - we should do a test insertion for this.

Reply:
RPS has no regressions (other than existing one of loading wer.dll, caused by https://github.com/dotnet/roslyn/pull/46703)